Workshop - 0.2 to 0.3 code transform of examples

shaileshgrowisto opened the pull request.

shaileshgrowisto committed:
0.2 to 0.3 code transform

amitu commented:
@shaileshgrowisto, thanks for the pull request. Regarding comments, we have moved to double semicolon for comments instead of single semicolons. Can you use that and update the PR?

shaileshgrowisto committed:
single semicolon comments converted to new format in double semicolon comments

amitu commented:
@shaileshgrowisto is this done? Should I merge it?

shaileshgrowisto commented:
> @shaileshgrowisto is this done? Should I merge it?

I have added the double semicolons, you can merge the code.

Also, for the Link feature, i updated with new release done by arpita but does not seem to work

Arpita-Jaiswal committed:
Minor fixes

amitu commented on 01-data/01-hello-world/FPM.ftd:
```diff
@@ -0,0 +1,4 @@
+;; check: https://ftd.dev/workshop/hello-world/ for instructions
+-- import: fpm
+
+-- fpm.package: hello
```
This file should not be checked in, we ask learner to create this file as first task.

shaileshgrowisto replied:
Oh, okay. I will remove this

amitu commented on 01-data/07-http-processor/index.ftd:
```diff
@@ -15,25 +15,25 @@ url: https://kameri-service.herokuapp.com/api/sample/
---- todo-view:
-t: $t
-$loop$: $todo-list as $t
+/-- todo-view:
+/t: $t
+/$loop$: $todo-list as $t
```
These should not be commented out.

amitu commented on 01-data/07-http-processor/index.ftd:
```diff
@@ -15,25 +15,25 @@ url: https://kameri-service.herokuapp.com/api/sample/
---- ftd.integer: $t.id
---- ftd.text: $t.status
---- ftd.text: $t.description
+/Uncomment the above code and run to see the below error
+/FPM-Error: path: /, PackageError { message: "failed to parse ParseError { message: \"hello/: not found (\\\"http\\\"), f: Variable::from_ast\", doc_id: \"hello/\", line_number: 11 }" }
```
Is this still happening?

AbrarNitk commented:
@amitu @shaileshgrowisto I merging this PR for now, please create a new PR.
